charts/telemetry-collector/values.yaml

```diff
@@ -0,0 +1,165 @@
+global: {}
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+bootstrap:
+  create: true
+  name: ""
+  kafka:
+    enabled: true
+    hostname: drax-kafka
+    port: 0
+  nats:
+    enabled: true
+    hostname: drax-nats
+    port: 0
+
+image:
+  repository: accelleran/telemetry-collector
+  pullPolicy: IfNotPresent
+  tag: ""
+
+imagePullSecrets:
+  - name: accelleran-secret
+
+accelleranLicense:
+  enabled: true
+  secretName: "accelleran-license"
+
+podSecurityContext:
+  {}
+  # fsGroup: 2000
+
+securityContext:
+  capabilities:
+    drop:
+      - ALL
+  # readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 10000
+
+resources:
+  {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+service:
+  enabled: true
+  name: ""
+  type: ClusterIP
+  ports:
+    udp:
+      port: 55555
+      targetPort: 55555
+      protocol: UDP
+
+influxdbAuthSecret: "{{ $.Release.Name }}-influxdb2-auth"
+
+config:
+  internal:
+    kafka:
+      url: ""
+    influxDB:
+      url: ""
+      token: ""
+      org: ""
+  pipelines:
+    - name: "cu_measurements"
+      component: "cu"
+      vendor: "accelleran"
+      enabled: "true"
+      consumers:
+        natsConsumer:
+          url: ""
```

Comment:
We probably don't want the drax chart to fill in the full pipelines config. As that will be needed because this is a list and thus it doesn't merge.

So I would maybe add an option `accelleranCu.nats.url` (bad name now) or something that we then inject here (and below for all occurrences) with templating.

Reply:
Won't we have the merging problem anyways the moment we ll want to add a new pipeline from the drax helm charts? I also don't find it a bad idea to inject the full config from the drax chart. It give a nice overview of which data pipelines and how are being deployed.

Reply:
True, hadn't even thought of us wanting to do that. Resolved then